docs/src/rules/id-denylist.md

```diff
@@ -38,6 +38,8 @@ For example, to restrict the use of common generic identifiers:
 }
 ```
 
+**Note:** First element of the array specifies the severity of the rule and it should be one of the following: 0 = "off", 1 = "warn", 2 = "error". Rest are the disallowed identifiers you want to add.
```

Comment:
Instead of restating the values for severity, can we just link to the section on configuring rules in the docs?

Reply:
Hi Nicholas, thanks for the feedback! I updated it. Please let me know if I can do further improvement in the sentence.